ingestion/src/metadata/cli/backup.py

```diff
@@ -98,13 +98,56 @@ def upload_backup(endpoint: str, bucket: str, key: str, file: Path) -> None:
         raise err
 
 
+def upload_backup_azure(account_url: str, container: str, file: Path) -> None:
+    """
+    Upload the mysqldump backup file.
+
+    :param account_url: Azure account url
+    :param container: Azure container to upload file to
+    :param file: file to upload
+    """
+
+    try:
+        from azure.identity import DefaultAzureCredential
+    except ModuleNotFoundError as err:
+        logger.debug(traceback.format_exc())
+        logger.error(
+            "Trying to import DefaultAzureCredential to run the backup upload."
+        )
+        raise err
+
+    click.secho(
+        f"Uploading {file} to {account_url}/{container}...",
+        fg="bright_green",
+    )
+
+    try:
+        # Create a blob client using the local file name as the name for the blob
+        blob_client = blob_service_client.get_blob_client(container=container, blob=file.name)
+
+        # Upload the created file
+        with open(file=file.absolute, mode="rb") as data:
```

Comment:
here we might be missing the `encoding`. The linting on the CI will fail. You can set `encoding=UTF_8`, we have a constant somewhere you can import